pkg/controller/dgraph/models/container.go

```diff
@@ -100,11 +110,11 @@ func StoreContainerProcessEdge(containerXID string, procsXIDs []string) error {
 	for _, procXID := range procsXIDs {
 		procUID := dgraph.GetUID(procXID, IsProc)
 		if procUID != "" {
-			procs = append(procs, &Proc{ID: dgraph.ID{UID: procUID}})
+			procs = append(procs, &Proc{ID: dgraph.ID{UID: procUID, Xid: procXID}})
```

Comment:
we need to soft delete non existing procs when we are persisting procs.

Reply:
Yes, I was thinking if we should hard delete or soft delete a process node. Will update timestamps for them too.

Reply:
Done